doc/source/user_guide/style.ipynb

```diff
@@ -116,22 +198,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from IPython.display import HTML\n",
-    "style = \\\n",
-    "\"<style>\"\\\n",
-    "\".table-cls {color: grey;}\"\\\n",
-    "\".cls1 {background-color: red; color: white;}\"\\\n",
-    "\".cls2 {background-color: blue; color: white;}\"\\\n",
-    "\".cls3 {font-weight: bold; font-style: italic; font-size:1.8em}\"\\\n",
-    "\"</style>\"\n",
-    "HTML(style)"
+    "with pd.option_context('display.precision', 2):\n",
+    "    html = df.style"
```

Comment:
this seems slightly confusing as you don't appear to use html (hard to tell though)